test/metabase/api/dashboard_test.clj

```diff
@@ -690,7 +693,335 @@
             (is (not= (:entity_id dashboard) (:entity_id response))
                 "The copy should have a new entity ID generated")
             (finally
-              (db/delete! Dashboard :id (u/the-id response)))))))))
+              (db/delete! Dashboard :id (u/the-id response))))))))
+  (testing "Deep copy: POST /api/dashboard/:id/copy"
+    (mt/dataset sample-dataset
+      (mt/with-temp* [Collection [source-coll {:name "Source collection"}]
+                      Collection [dest-coll   {:name "Destination collection"}]
+                      Dashboard  [dashboard {:name          "Dashboard to be Copied"
+                                             :description   "A description"
+                                             :collection_id (u/the-id source-coll)
+                                             :creator_id    (mt/user->id :rasta)}]
+                      Card       [total-card  {:name "Total orders per month"
+                                               :collection_id (u/the-id source-coll)
+                                               :display :line
+                                               :visualization_settings
+                                               {:graph.dimensions ["CREATED_AT"]
```

Comment:
nit: Correct me if I'm wrong but  `display` and `visualization_settings` are not used in the tests, so maybe we could delete them? 
It makes the tests more focused IMO.

Reply:
fair point. I set this up so that the dashboard is actually a well-formed dashboard. If you pause the tests halfway through running the cards have line graphs and some are stacked multi-series. certainly true there are no assertions on the settings at the moment. I'll think about that and see if i can/should assert on that.